idaes/power_generation/carbon_capture/compression_system/tests/test_compressor.py

```diff
@@ -114,10 +115,10 @@ def test_run(build_unit):
                       * m.fs.unit.outlet.enth_mol[0]
                       + m.fs.unit.work_mechanical[0]))
     # pressure change
-    assert (pytest.approx(177991.4247463311, abs=0.1) ==
+    assert (pytest.approx(185822.29121655718, abs=0.1) ==
```

Comment:
If you are setting an absolute tolerance of 0.1, you don't need all the additional significant figures in the test value.